exchange/bidder_test.go

```diff
@@ -660,9 +663,9 @@ func TestMultiCurrencies_RateConverterNotSet(t *testing.T) {
 		)
 
 		// Verify:
-		assert.Equal(t, false, (seatBid == nil && tc.expectedBidsCount != 0), fmt.Sprint("Case:", strings.Join(tc.bidCurrency, ",")))
-		assert.Equal(t, tc.expectedBidsCount, uint(len(seatBid.bids)), fmt.Sprint("Case:", strings.Join(tc.bidCurrency, ",")))
-		assert.ElementsMatch(t, tc.expectedBadCurrencyErrors, errs, fmt.Sprint("Case:", strings.Join(tc.bidCurrency, ",")))
+		assert.Equal(t, false, (seatBid == nil && tc.expectedBidsCount != 0), fmt.Sprint("Case:", strings.Join(tc.bidCurrency, ",")), tc.description)
```

Comment:
Like this approach of adding the descriptions to the table test cases. One question though, since the descriptions are now added, do we still need the existing error messages: `fmt.Sprint("Case:", strings.Join(tc.bidCurrency, ","))` ?

Reply:
Actually, you are right, we don't need it any longer, I can remove it :)
Thanks !

Reply:
Just updated the PR, thanks @mansinahar :)

Reply:
Awesome. Thanks @benjaminch !